data-raw/paper/paper.Rmd

```diff
@@ -0,0 +1,73 @@
+
+The principled and generalized approach of the package lends itself to diverse applications of predictive modeling. This capability, for one, was recognized with the receipt of the American Statistical Association's 2021 John M. Chambers Award, awarded "for the development and implementation of computational tools for the statistical profession by a graduate or undergraduate student" [@chambers]. The package's functionality has also been shared in venues such as R/Pharma 2020, rstudio::global(2021), and the 2021 Joint Statistical Meetings. To date, the package has been downloaded more than 10,000 times, evidencing its key contribution to a software ecosystem utilized in diverse research contexts.
+
+# Underlying Principles
+
+At a high level, stacked ensemble models in {stacks} are formed from *model definitions*, which specify a set of instructions to fit a model or set of models using {parsnip}, {recipes}, and {workflows}. Each model specified in a model definition is referred to as a *candidate member* in the package. To be used in the same model stack, all model definitions must share a *resampling object*, as defined in the {rsample} package, such as a cross-fold validation or set of bootstrap samples. After initializing a `data_stack` object with the `stacks()` function, model definitions can be iteratively added to the data stack with `add_candidates()`. This function collates the predictions on the assessment set, a subset of the training data used for preliminary model validation, from each candidate specified in the model definitions to the data stack.
+
+```{r, echo = FALSE, out.width = '100%'}
+knitr::include_graphics("class_data_stack.png")
+```
+
+The above diagram represents a data stack containing 11 candidate members, where 4 come from a shared K-nearest neighbors model definition, 1 arises from a linear model, and 6 come from a shared support vector machine model definition. Model definitions can result in multiple candidates when several possible hyperparameter values are being optimized over (e.g. the number of neighbors $k$ or the precision parameter for the support vector machine's radial basis function).
+
+After all candidates are collated to the data stack, the user can fit the meta-learner using the `blend_predictions()` function, which fits an elastic net model on the data stack, predicting the true outcome using the predictions from each of the candidate members. The coefficients of this model form the *stacking coefficients*, which are the weightings for each of the member models in ultimately predicting the outcome. Candidate members with non-zero stacking coefficients are *members*, and must be fitted on the entire training set with the `fit_members()` function. This function outputs a `model_stack` object, and is ready to predict on new data.
```

Comment:
I would add a reference for the elasticnet model. Maybe [this reference](https://hastie.su.domains/StatLearnSparsity/index.html)

Reply:
You got it👍 [ab2b729](https://github.com/tidymodels/stacks/pull/98/commits/ab2b729086fa36c0ed1667aff1147c08e6c7be8c)